pkg/client/kubernetes/apply.go

```diff
@@ -188,44 +188,64 @@ func (c *Applier) mergeObjects(newObj, oldObj *unstructured.Unstructured, mergeF
 // all concatenated in a byte slice, and sends them one after the other to the API server. If a resource
 // already exists at the API server, it will update it. It returns an error as soon as the first error occurs.
 func (c *Applier) ApplyManifest(ctx context.Context, r UnstructuredReader, options ApplierOptions) error {
+	errList := make([]error, 0)
+
 	for {
 		obj, err := r.Read()
 		if err == io.EOF {
-			return nil
+			break
 		}
 		if err != nil {
-			return err
+			errList = append(errList, fmt.Errorf("could not read object: %+v", err))
+			continue
 		}
 		if obj == nil {
 			continue
 		}
 
 		if err := c.applyObject(ctx, obj, options); err != nil {
-			return err
+			errList = append(errList, fmt.Errorf("could not apply object of kind '%s' '%s/%s': %+v", obj.GetKind(), obj.GetNamespace(), obj.GetName(), err))
```

Comment:
Nit: Use `multierror.Append(errs, err)` instead of error slice.

Reply:
Nice suggestion, done :)
PTAL